source/encryption.rst

```diff
@@ -0,0 +1,132 @@
+.. _encryption:
+
+Data at rest encryption
+************************
+
+`Ful data at rest encryption in Percona XtraDB Cluster <https://www.percona.com/doc/percona-xtradb-cluster/LATEST/management/data_at_rest_encryption.html>`_ is supported by the Operator since version 1.4.0.
+
+..note:: "Data at rest" means inactive data stored as files, database records, etc.
```

Comment:
We could link to an outside resource here such as Wikipedia if we'd like.  Not required though.